monitor-e2e/test/common.js

```diff
@@ -0,0 +1,87 @@
+const assert = require('assert')
+const axios = require('axios')
+const { nativeToErcBridge, ercToErcBridge, ercToNativeBridge } = require('../../e2e-commons/constants.json')
+
+const types = [
+  { description: 'NATIVE TO ERC', baseUrl: nativeToErcBridge.monitor },
+  { description: 'ERC TO ERC', baseUrl: ercToErcBridge.monitor },
+  { description: 'ERC TO NATIVE', baseUrl: ercToNativeBridge.monitor }
+]
+
+types.forEach(type => {
+  describe(type.description, () => {
+    describe('balances', async () => {
+      let data
+
+      before(async () => {
+        ;({ data } = await axios.get(`${type.baseUrl}`))
+      })
+
+      describe('home', async () => {
+        it('should contain deposits:', () => assert(data.home.deposits === 0))
+        it('should contain withdrawals', () => assert(data.home.withdrawals === 0))
+      })
+
+      describe('foreign', async () => {
+        it('should contain deposits:', () => assert(data.foreign.deposits === 0))
+        it('should contain withdrawals', () => assert(data.foreign.withdrawals === 0))
+      })
+
+      describe('general', async () => {
+        it('should contain balanceDiff', () => assert(data.balanceDiff === 0))
```

Comment:
it also needed to have a test to check that the monitor catches situations when the bridge is unbalanced since a deposit or withdrawal operation is stuck.

Reply:
This was not a small change, so I created a separate PR targeting this PR, so it can be reviewed separately. Please take a look: https://github.com/poanetwork/tokenbridge/pull/191